### PR TITLE
ci: add coveralls

### PR DESCRIPTION
**Pull request by acud**
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1956)

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -66,6 +66,14 @@ jobs:
...
       env:
         DEEPSOURCE_DSN: ${{ secrets.DEEPSOURCE_DSN }}
+    - name: Coveralls
+      if: matrix.os == 'ubuntu-latest'
+      uses: shogo82148/actions-goveralls@v1
+      with:
+        path-to-profile: cover.out
+      env:
+        COVERALLS_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+
```
Comment (vandot) [inline .github/workflows/go.yml:76]: I would remove this line, there is no empty line in whole job definition

**Review by vandot — APPROVED** [review]

**Review by mrekucci — COMMENTED** [review]
Does this actually work? When I check the tree, the batch says `coverage|unknown`: https://github.com/ethersphere/bee/tree/coveralls

> Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1956#-:-MbCbpd_2QWBrolS3E2H:b14jf9z)* status: all files reviewed, 1 unresolved discussion (waiting on @acud)

**Review by acud — COMMENTED** [review]
my guess is that because the badge refers to the repo it probably tries to get the coverage for `master`. since this is still not on `master`, it probably won't work until it runs on `master`

> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1956#-:-MbCeT2BCpZxfW1asPxQ:b-tmk4fm)* status: all files reviewed, 1 unresolved discussion (waiting on @vandot)

___
*[.github/workflows/go.yml, line 76 at r1](https://reviewable.io/reviews/ethersphere/bee/1956#-MbC7AzeEq1zF_Isewb6-r1-76:-MbCeQdRB0vFgnDYobzA:b-896fix) ([raw file](https://github.com/ethersphere/bee/blob/553812b93b9e6e0daaf46bdffe422b3e0811677a/.github/workflows/go.yml#L76)):*
<details><summary><i>Previously, vandot (Ivan Vandot) wrote…</i></summary><blockquote>

I would remove this line, there is no empty line in whole job definition
</blockquote></details>

Done.

**Review by esadakar — APPROVED** [review]
nice...

**Pull request merged**